src/web/pages/alerts/dialog.jsx

```diff
@@ -717,6 +717,7 @@ class AlertDialog extends React.Component {
               {values.method === METHOD_TYPE_SCP && (
                 <ScpMethodPart
                   prefix="method_data"
+                  capabilities={capabilities}
```

Comment:
You can skip passing `capabilities` here for all the methods if you will go with the hook.
```suggestion
```